Sources/SwiftCompilerPluginMessageHandling/Macros.swift

```diff
@@ -234,3 +235,17 @@ extension CompilerPluginMessageHandler {
     )
   }
 }
+
+fileprivate extension SyntaxProtocol {
+  /// Perform a format if required and then trim any leading/trailing
+  /// whitespace.
+  func formattedExpansion(_ mode: FormatMode) -> String {
+    let formatted: Syntax
+    if mode == .auto {
```

Comment:
`switch` so we don’t forget to add new branches if we add new cases to `FormatMode`?